db-client/src/index.ts

```diff
@@ -41,9 +42,29 @@ const db = new DB(
     agent: dbURL.startsWith('https://')
       ? new https.Agent({ keepAlive: true })
       : new http.Agent({ keepAlive: true }),
+    ...options,
   },
 );
 
+let db = makeDb();
+
+/**
+ * Set HTTP options for DB requests
+ *
+ * ## Example: increase timeouts for large queries
+ *
+ * ```js
+ * db.setDbOptions({ timeoutMs: 10000 });
+ * ```
+ */
+export function setDbOptions(options: Options = {}) {
+  if (options.timeoutMs !== undefined &&
+    (typeof options.timeoutMs !== 'number' || options.timeoutMs >= 60000 || options.timeoutMs <= 0)) {
+    throw new Error('Resshuffle DB configuration error: timeoutMs must be a number between 0 and 60000');
```

Comment:
Nit: "Reshuffle".

Reply:
Why an upper limit? 1 minute is plenty, but so are 30 seconds or 2 minutes.

Reply:
Thanks!

Added comment about the limit to hint about the suggested usage (we didn't have to limit, but it would be a surprise why queries are getting lost).